### PR TITLE
Include public workspaces

### PR DESCRIPTION
**Pull request by elizzhao**
adds a checkbox on the workspaces page that includes public workspaces into the list.

### CODE REVIEW

File: src/pages/workspaces/List.js [hunk 1]
```diff
@@ -245,9 +247,15 @@ export const WorkspaceList = _.flow(
...
       h(PageBox, { style: { position: 'relative' } }, [
-        div({ style: { display: 'flex', alignItems: 'center', justifyContent: 'space-between', marginBottom: '1rem' } }, [
+        div({ style: { display: 'flex', alignItems: 'center', justifyContent: 'flex-end', marginBottom: '1rem' } }, [
           div({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase' } }, ['Workspaces']),
-          div({ style: { marginLeft: 'auto', flex: '0 0 300px' } }, [
+          div({ style: { marginLeft: 'auto' } }, [
+            h(LabeledCheckbox, {
+              checked: includePublic === true,
+              onChange: v => this.setState({ includePublic: v })
+            }, ' Include public workspaces')
```
Comment (katevoss) [inline src/pages/workspaces/List.js]:
```suggestion
            }, ' Show public workspaces')
```

**Review by katevoss — APPROVED** [review]
Looks good! One nit is that clicking on the text also triggers the checkbox on/off. 
Screen Shot 2019-04-04 at 10 14 14 AM

Also, when you navigate away from the workspace list, "include public workspaces" check does not persist. The filter by access level/project does persist. Not a concern, just a behavior to note.

**Comment by elizzhao** [comment]
@katevoss I think clicking on the text to check the box is the standard behavior we have

File: src/pages/workspaces/List.js (hunk 1 above)
Comment (elizzhao) [reply src/pages/workspaces/List.js]: Nice catch on the persisting! I forgot to save it to state history. I like that the behavior matches now

**Pull request merged**